### PR TITLE
Migrate Vue component styling into Sass files

### LINKED ISSUE #17: Move styling out of Vue files and into Sass files
All styling should be found in sass files to make changes easier to find.

### PR DESCRIPTION
**Pull request by Engineering-Robert**
Fixes #17

### CODE REVIEW

File: resources/assets/sass/pages/_home.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.navigation-drawer .router-link-exact-active {
```
Comment (Voziv) [inline resources/assets/sass/pages/_home.scss]: doesnt belong in home

**Comment by Engineering-Robert** [comment]
Alright, I've updated this @Voziv

**Comment by Voziv** [comment]
Just testing slack

**Comment by Engineering-Robert** [comment]
@Voziv conflicts are now resolved.

**Pull request merged**